ingress/controllers/nginx/nginx/main.go

```diff
@@ -67,7 +67,9 @@ func NewManager(kubeClient *client.Client) *Manager {
 
 	ngx.sslDHParam = ngx.SearchDHParamFile(config.SSLDirectory)
 
-	ngx.loadTemplate()
+	if err := ngx.loadTemplate(); err != nil {
+		glog.Fatalf("invalid NGINX template: %v", err)
```

Comment:
so we don't watch the custom template? failing on first failure to load is fine, I'm more concerned with what happens if i:
1. kubectl exec and overwrite template
2. endpoints change

do we crashloop, or keep using the old template (sounds like we keep using the old template, meaning we ignore updates to the config map. that's fine as long as it's clear in the doc)?